src/azul/azulclient.py

```diff
@@ -412,16 +412,12 @@ def reset_indexer(self,
             for catalog in catalogs:
                 self.create_all_indices(catalog)
 
-    def wait_for_indexer(self, **kwargs):
+    def wait_for_indexer(self):
         """
         Wait for indexer to begin processing notifications, then wait for work
         to finish.
-
-        :param kwargs: keyword arguments to Queues.wait_for_queue_level when
-                       waiting for work to finish.
         """
-        self.queues.wait_for_queue_level(empty=False)
-        self.queues.wait_for_queue_level(empty=True, **kwargs)
+        self.queues.wait_to_stabilize(empty=True)
```

Comment:
This seems to be the only call site, so if `empty` is always True we should eliminate the argument and simplify the method.